### PR TITLE
Adding log source latency stats to InfoProvider 

### PR DESCRIPTION
**Pull request by DDuongNguyen**
### What does this PR do?
Getting rid of log latency from source but instead push them to InforProvider to centralize all log attributes.

### Motivation

Creating a centralized place so logs attribute can be rendered from 1 singular place

### Additional Notes

Still working on testing actual log files to see if the latency is accurate 

### Describe how to test/QA your changes
- Render both Agent Status and Agent GUI to see if log latency stats would show up from InfroProvider interface.  
- Check if log latency stats are accurate.

### CODE REVIEW

File: pkg/util/stats_tracker.go [hunk 1]
```diff
@@ -161,3 +162,21 @@ func (s *StatsTracker) dropOldPoints(now int64) {
...
+}
+
+func (s *StatsTracker) Info() []string {
+	AllTimeAvgLatency := s.AllTimeAvg() / int64(time.Millisecond)
+	AllTimePeakLatency := s.AllTimePeak() / int64(time.Millisecond)
+	RecentAvgLatency := s.MovingAvg() / int64(time.Millisecond)
+	RecentPeakLatency := s.MovingPeak() / int64(time.Millisecond)
+
+	return []string{
+		"Average Latency (ms): " + strconv.FormatInt(RecentAvgLatency, 10),
```
Comment (gh123man) [inline pkg/util/stats_tracker.go:177]: NIT: I'd prefer using `SPrintF` over `FormatInt`
```suggestion
		fmt.Sprintf("Average Latency (ms): %d", RecentAvgLatency)
```
(as well as for the others below)

File: pkg/util/stats_tracker.go [hunk 2]
```diff
@@ -161,3 +162,21 @@ func (s *StatsTracker) dropOldPoints(now int64) {
 		s.peakPointsHead = nil
 	}
 }
+
+func (s *StatsTracker) InfoKey() string {
+	return "Latency Stats"
```
Comment (gh123man) [inline pkg/util/stats_tracker.go:167]:
```suggestion
	return "Pipeline Latency"
```

I think this is slightly more descriptive of what the metrics mean.

File: pkg/logs/status/status.go [hunk 3]
```diff
@@ -37,16 +37,16 @@ var (
...
-	AllTimePeakLatency int64                  `json:"all_time_peak_latency"`
-	RecentAvgLatency   int64                  `json:"recent_avg_latency"`
-	RecentPeakLatency  int64                  `json:"recent_peak_latency"`
-	Type               string                 `json:"type"`
-	Configuration      map[string]interface{} `json:"configuration"`
-	Status             string                 `json:"status"`
-	Inputs             []string               `json:"inputs"`
-	Messages           []string               `json:"messages"`
-	Info               map[string][]string    `json:"info"`
+	// AllTimeAvgLatency  int64 `json:"all_time_avg_latency"`
```
Comment (gh123man) [inline pkg/logs/status/status.go:40]: Can remove these commented lines.

**Review by gh123man — CHANGES_REQUESTED** [review]
Just some minor comments. Overall looks great.

**Review by gh123man — APPROVED** [review]

**Review by Kaderinho — APPROVED** [review]

**Pull request merged**